### ISSUE TITLE
Request for user-definable FPM (--package) parameters

### ISSUE THREAD

**Issue by DirkdeDraak** [hpcugent/easybuild#183]
This is a request to be able to have user-definable FPM parameters.

I now patch  easybuild/tools/package/utilities.py to have the following parameters for FPM included:
```
+        '--prefix', '""', # not relocatable
+        '--maintainer', "my@mail.com",
+        '--vendor', "my@mail.com",
```

**Comment by boegel** [comment]
@rjeschmi: yours?

**Comment by rjeschmi** [comment]
Yes, I can look at this.

Is the idea of the prefix request to remove that option so that the package is expressly not relocatable?

I have looked at relocation a bit and wonder whether it is a good feature. Some parts of relocation are easy since we load the dependencies using LD_LIBRARY_PATH, but others would be a bit difficult since it would need to be handled at build time (like setting the shebang path for perl in some scripts included with perl).

Maintainer and vendor are both good options.

**Comment by DirkdeDraak** [comment]
I added the prefix="" to make sure the rpm will not be installed on another path, while files may hold hardcoded paths.
But it should be at the users discretion to define it.

I didn't want to push a standard here.
The idea for my request is to have fully user-definable values for FPM.
So the user can choose which values to set/override.

**Comment by rjeschmi** [comment]
Yeah, fair enough. I think it is fine to give the users the options. The default for prefix makes sense.

Thanks for the tips
